src/ui-kit/form-controls/sam-sds-autocomplete/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -445,11 +448,7 @@ export class SAMSDSAutocompleteSearchComponent implements ControlValueAccessor {
    * @param message 
    */
   private addScreenReaderMessage(message: string) {
-    const srResults: HTMLElement = document.createElement('li');
-    srResults.innerText = message;
-    if (this.srOnly && this.srOnly.nativeElement) {
-      this.srOnly.nativeElement.appendChild(srResults);
```

Comment:
This was the cause of additional whitespace being generated at the bottom of the page as user hovers through autocomplete results. We continuously append list item element to screen reader section as new results are made available / user hovers over autocomplete items. This causes the number fo list items to continuously increase and take up more space. The fix adds a single li element to html and uses data binding to update screen reader message